qiskit/backends/ibmq/ibmqjob.py

```diff
@@ -297,9 +278,9 @@ def running(self):
             bool: True if job is running, else False.
 
         Raises:
-            QISKitError: couldn't get job status from server
+            JobError: couldn't get job status from server
         """
-        return self.status['status'] == JobStatus.RUNNING
+        return self.status() == JobStatus.RUNNING
```

Comment:
You made a good point, in order to avoid possible user code patterns that could derive in sending too much requests, we should be removing all state properties and leave `status()` method as the only one needed to query the real status of the Job.